float64[] xD
float64[] xD1Dot
float64[] xD2Dot
float64[] xD3Dot
float64[] xD4Dot
float64[] b1D
float64[] b1D1Dot
float64[] b1D2Dot
